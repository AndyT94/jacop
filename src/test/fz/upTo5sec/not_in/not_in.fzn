var 2..2: v:: output_var;
array [1..2] of var int: values:: output_array([1..2]) = [1,3];
solve  satisfy;
