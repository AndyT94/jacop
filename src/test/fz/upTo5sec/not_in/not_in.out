v = 2;
values = array1d(1..2, [1, 3]);
----------